src/types/election/published.ts

```diff
@@ -52,7 +67,14 @@ export class PublishedElection extends Election {
     });
     this._id = params.id;
     this._organizationId = params.organizationId;
-    this._status = params.status;
+    switch (params.status) {
+      case ElectionStatusReady.READY:
+        this._status = this.startDate <= new Date() ? ElectionStatus.ONGOING : ElectionStatus.UPCOMING;
```

Comment:
@marcvelmer n the previous SDK?UI the comparison was done in dates but estimated from the blockNumbers, considering that they can be a source of truth, in case the client and vocdoni-node dates do not agree (very probable scenario). What is the logic of using directly dates here?

Reply:
We decided to not use the blocks in the API, therefore I cannot access this information. I think it's OK to use dates as they are calculated by the node based on the blocks